media/base/media_util.cc

```diff
@@ -31,8 +31,12 @@ AudioParameters::Format ConvertAudioCodecToBitstreamFormat(AudioCodec codec) {
 }
 
 bool MediaTraceIsEnabled() {
+#if defined(STARBOARD)
```

Comment:
Let's create a bug to revisit if this can be enabled.

Reply:
We can likely enable it now. Trace events haven't yet been updated in base (`TRACE_EVENT_CATEGORY_GROUP_ENABLED` is undefined), though it looks like a workaround is in place.
https://github.com/youtube/cobalt/blob/a31a154f0072cadb5958c44cfc1731130d2e6d20/base/trace_event/trace_event.h